src/main/charts/confluence/templates/_fluentd_templates.tpl

```diff
@@ -0,0 +1,37 @@
+{{- define "fluentd.container" -}}
+{{ if .Values.fluentd.enabled }}
+- name: fluentd
+  image: {{ .Values.fluentd.imageName }}
+  command: ["fluentd", "-c", "/fluentd/etc/fluent.conf", "-v"]
+  volumeMounts:
+    - name: local-home
+      mountPath: /application-data
+      readOnly: true
```

Comment:
Mount the pod's `local-home` volume so that fluentd has access to the application's log files.

Reply:
Do we need to mount an entire confluence home? Does `/application-data/logs` make more sense? Also, our default is https://github.com/atlassian-labs/data-center-helm-charts/blob/master/src/main/charts/confluence/values.yaml#L200. Perhaps it makes sense to use `{{ .Values.volumes.localHome.persistentVolumeClaim.mountPath }}` rather than hardcode the path since we leave it configurable anyway?

Reply:
> Do we need to mount an entire confluence home? Does /application-data/logs make more sense?

Good idea. I've added a `subPath` to the `volumeMount` so that only the logs subdir is mounted.

As for the mount path itself, we *could* use the same path as the confluence container, but I don't think it adds anything, and it suggests that they need to be the same, when in fact they don't.